test/clojure/clojurewerkz/cassaforte/cql_test.clj

```diff
@@ -321,7 +321,9 @@
     (let [xs  (select *session* :events
                       (unix-timestamp-of :created_at)
                       (limit 5))
-          ts' (get (first xs) (keyword "unixTimestampOf(created_at)"))]
+          ts' (or
+                (get (first xs) (keyword "unixTimestampOf(created_at)"))
+                (get (first xs) (keyword "system.unixtimestampof(created_at)")))]
```

Comment:
C* 3.0 changed the way it names columns for functions, which means it might be a breaking change for users.